### ISSUE TITLE
How does CodeGeex2 compare with GPT-3.5 on multilingual code generation?

### ISSUE THREAD

**Issue by Symbolk** [THUDM/CodeGeeX2#68]
Hi, I noticed that the comparison in README does not contain the OpenAI models and Claude, since they all provide API  and actually serves as the backend for Copilot, do you have the plan to add these numbers?

**Comment by Stanislas0** [comment]
> Hi, I noticed that the comparison in README does not contain the OpenAI models and Claude, since they all provide API and actually serves as the backend for Copilot, do you have the plan to add these numbers?

Yeah, it's possible to add them in the future. I imagine there is still a large gap between close-sourced and open-sourced models.

**Comment by ishaan-jaff** [comment]
Hi @Stanislas0 @Symbolk I'm the maintainer of liteLLM - https://github.com/BerriAI/litellm/
We allow you to call GPT-3.5 (and 50+ LLMs including Hugging face, Ollama (local llama2)) using OpenAI chatGPT inputs/outputs.

I'd love to help make an integration if this is useful for this issue !😊
